subworkflows/nf-core/deepvariant/tests/main.nf.test

```diff
@@ -1,117 +1,104 @@
-nextflow_process {
+nextflow_workflow {
 
-    name "Test Process DEEPVARIANT"
+    name "Test Subworkflow DEEPVARIANT"
     script "../main.nf"
-    process "DEEPVARIANT"
+    workflow "DEEPVARIANT"
+
+    tag "subworkflows"
+    tag "subworkflows_nfcore"
+    tag "subworkflows/deepvariant"
 
     tag "deepvariant"
-    tag "modules"
-    tag "modules_nfcore"
+    tag "deepvariant/makeexamples"
+    tag "deepvariant/callvariants"
+    tag "deepvariant/postprocessvariants"
 
-    test("homo_sapiens - [bam, bai] - fasta - fai") {
+    test("homo_sapiens - two inputs - bam - fasta - fai") {
```

Comment:
This test and the one below only runs `test` and not `test2`. Not sure if removing Channel.of, or adding collect() to the rest of the inputs is the way to go.